### DISCUSSION TITLE
How to recognize a repository as the package on pypi?

### DISCUSSION THREAD

**Discussion by cadop** [community/community#37525] [Other Feature Feedback, Questions, & Ideas] [answered]
I have two repositories.  One relies on the python package (RA) that is based on the second repository (RB).

RA correctly recognizes it has a dependency on a package with the name of RB (but does not provide a link), and has a dropdown of the dependencies of the RB package. 

RB does not recognize RA is a dependent.  

I think this may happen because RB contains the source of a C API with Python and C# bindings, and is all just contained in a src folder.  

Do I need to have an action setup that directly pushes to pypi, or what is the _thing_ that triggers a connection between the pypi repo and the github repo?

RA = https://github.com/cadop/ov_dhart
RB = https://github.com/cadop/dhart

Thanks!

**Comment by cadop** [comment, marked as answer]
The issue was that I didn't have the pypi page pointing back to the github page. 

So specifically, the package that is a dependency just needs to have in the `setup` or `toml` the URL set so that on pypi the `Homepage` points to the github page itself. Then the hyperlink works.

**Comment by Heviwulansari** [comment]
[> ](https://www.icmarkets.com/global/id/trading-markets/range-of-markets)
